packages/opentelemetry-core/src/common/time.ts

```diff
@@ -64,19 +66,27 @@ export function hrTime(performanceNow?: number): api.HrTime {
  * @param time
  */
 export function timeInputToHrTime(time: api.TimeInput): api.HrTime {
-  // process.hrtime
+  return nanosToHrTime(timeInputToNano(time));
```

Comment:
maybe we could keep the type check to avoid unnecessary calculation

```suggestion
  if (isTimeInputHrTime(time)) {
    return time as api.HrTime;
  }
  return nanosToHrTime(timeInputToNano(time));
```

Reply:
I can add it back but the function this calls has the check still